.github/workflows/codeql.yml

```diff
@@ -23,20 +23,27 @@ jobs:
       fail-fast: false
       matrix:
         language: [ 'java' ]
+
     steps:
     - name: Checkout repository
       uses: actions/checkout@v3
 
+    - name: Setup Java JDK
```

Comment:
If we setup now java 17, what java version was used before?

Reply:
by default still uses java 11 if not set explicitly